index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Anthony Martin Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <main>
+        <header>
+            <div class="header-logo">Anthony Martin</div>
+            <div class="header-menu-button">
+                <div></div>
+                <div></div>
+                <div></div>
+            </div>
+        </header>
+
+        <section class="headline">
+            <h1 class="headline_title">I’m Anthony
+                Glad to see you!</h1>
+            <p class="headline_description">I’m a software developer!
+                I can help you build a product , feature or website Look
+                through some of my work and experience! If you like
+                what you see and have a project you need coded, don’t
+                hestiate to contact me.</p>
+            <a href="https://wa.me/584142221022">Let’s connect</a>
+            <nav>
+                <ul class="headline_social">
+                    <li><a href="#"><img src="img/social1.svg" alt="Twitter"></a></li>
+                    <li><a href="#"><img src="img/social2.svg" alt="LinkedIn"></a></li>
+                    <li><a href="#"><img src="img/social3.svg" alt="Social M"></a></li>
+                    <li><a href="#"><img src="img/social4.svg" alt="Github"></a></li>
+                    <li><a href="#"><img src="img/social5.svg" alt="Another Social Media"></a></li>
+                </ul>
+            </nav>
```

Comment:
- For semantics reasons and to help improve your page's structure, I suggest you remove this section from the `nav` tag. Using the `ul` tag fro this section is enough. Please fix this 😄 😄